Jenkinsfile

```diff
@@ -32,6 +32,7 @@ pipeline {
     post {
         always {           
             archiveArtifacts(artifacts: 'output/', fingerprint: true)
```

Comment:
You can simplify it.
`archiveArtifacts(artifacts: 'output/**/*', fingerprint: true)`